File: Aulas/Aula04/principal.py
from dados_climaticos import Clima
lista = []

# um_clima = Clima("2020", "Janeiro", "Quente", "pouca")
# print("Exibindo dados de um mes...", obj_clima)
linha = "2020,Janeiro,Quente,muita"
dados_linha = linha.split(",")

obj_clima = Clima(dados_linha[0], dados_linha[1], dados_linha[2], dados_linha[3] )

if obj_clima not in lista:
    lista.append(obj_clima)
# print("Exibindo dados de um mes...", obj_clima)

linha = "2020,Janeiro,frio,pouca"
dados_linha = linha.split(",")

obj_clima = Clima(dados_linha[0], dados_linha[1], dados_linha[2], dados_linha[3] )

if obj_clima not in lista:
    lista.append(obj_clima)

for c in lista:
    print(c)

File: Aulas/Aula04/principal.java
import java.util.ArrayList;

import desafio.dadosClimaticos;

public class principal {
    public static void main(String[] args) {
        ArrayList<dadosClimaticos> lista = new ArrayList<>();

        // simulando que estamos lendo linha a linha do arquivo
        String linha = "2020,Janeiro,Quente,muita";
        // dadosClimaticos umClima = new dadosClimaticos("2020", "Janeiro", "Quente", "muita");
        String dadosLinha[] = linha.split(",");
        dadosClimaticos objDadosClimaticos = new dadosClimaticos(dadosLinha[0], dadosLinha[1], dadosLinha[2], dadosLinha[3]);


        linha = "2020,Janeiro,frio,pouca";
        dadosLinha = linha.split(",");
        objDadosClimaticos = new dadosClimaticos(dadosLinha[0], dadosLinha[1], dadosLinha[2], dadosLinha[3]);
        if(!lista.contains(objDadosClimaticos)){
            lista.add(objDadosClimaticos);

        }

        for(dadosClimaticos c : lista){
            System.out.println(c);
        }
        


        System.out.println("Exibindo dados de um mês: "+ objDadosClimaticos);

    }
    
}
